frontend/pages/sign-in/index.tsx

```diff
@@ -18,11 +18,10 @@ import ErrorMessage from 'components/forms/error-message';
 import Input from 'components/forms/input';
```

Comment:
The same here. I changed the Enums format, so I had to change all the files that they are used